src/term/mod.rs

```diff
@@ -506,7 +506,7 @@ impl Term {
         // Scroll up to keep cursor and as much context as possible in grid.
         // This only runs when the lines decreases.
         self.scroll_region = Line(0)..self.grid.num_lines();
-
+        
```

Comment:
ewe, guess I don't have my editor setup to remove trailing whitespace for rust files yet. Sorry!